src/test/java/net/minecraftforge/debug/MapLocatingVillage.java

```diff
@@ -0,0 +1,123 @@
+package net.minecraftforge.debug;
+
+import net.minecraft.client.gui.MapItemRenderer;
+import net.minecraft.client.renderer.block.model.ModelBakery;
+import net.minecraft.client.renderer.block.model.ModelResourceLocation;
+import net.minecraft.creativetab.CreativeTabs;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemMap;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.ActionResult;
+import net.minecraft.util.EnumActionResult;
+import net.minecraft.util.EnumHand;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.text.TextComponentString;
+import net.minecraft.world.World;
+import net.minecraft.world.storage.MapData;
+import net.minecraft.world.storage.MapDecoration;
+import net.minecraftforge.client.event.ModelRegistryEvent;
+import net.minecraftforge.client.model.ModelLoader;
+import net.minecraftforge.common.util.EnumHelper;
+import net.minecraftforge.event.RegistryEvent;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.SidedProxy;
+import net.minecraftforge.fml.common.event.FMLPreInitializationEvent;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+import net.minecraftforge.fml.common.registry.GameRegistry;
+
+@Mod(modid = MapLocatingVillage.MODID, name = "MapLocatingVillage", version = MapLocatingVillage.VERSION, acceptableRemoteVersions = "*")
+@Mod.EventBusSubscriber
+public class MapLocatingVillage
+{
+    public static final String MODID = "maplocatingvillage";
+    public static final String VERSION = "1.0";
+
+    public static final MapDecoration.Type VILLAGE_MAP_DECORATION = EnumHelper.addMapDecoration("VILLAGE", (byte) 0, true, -1);
+    public static final ResourceLocation VILLAGE_MAP_GIVER_NAME = new ResourceLocation(MODID, "village_map_giver");
+
+    @SidedProxy
+    public static CommonProxy PROXY;
+
+    @GameRegistry.ObjectHolder("village_map_giver")
```

Comment:
I don't think you need this value in the annotation since the field name has specified it.